documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -167,15 +167,45 @@ Ensure that the recreated secrets have exactly the same name, labels, and fields
 .. Apply the `KafkaUser` resource configuration:
 +
 [source,shell]
-kubectl apply -f <user_configuration_file>
+kubectl apply -f <user_configuration_file> -n my-project
 
-. Deploy the Kafka cluster using the original configuration for the `Kafka` resource:
+. Deploy the Kafka cluster using the original configuration for the `Kafka` resource.
+If it's a KRaft cluster, add the annotation `strimzi.io/pause-reconciliation="true"` to the YAML file. 
```

Comment:
This should be a KRaft step like the unpause.

Reply:
Yeah, let me split the procedures.